src/main/java/eu/donals/Location.java

```diff
@@ -0,0 +1,53 @@
+package eu.donals;
+
+import java.util.List;
+
+
+public class Location {
+    private String name;
+    private String description;
+    private NPC npc;
+    private List<String> neighbours;
+
+    public class NPC {
```

Comment:
I like the use of a nested class here, makes the structure of the all the getters and setters instantly clear.

Reply:
agree